influxdb/1.2/alpine/entrypoint.sh

```diff
@@ -1,8 +1,12 @@
-#!/bin/sh
+#!/bin/bash
 set -e
 
 if [ "${1:0:1}" = '-' ]; then
     set -- influxd "$@"
 fi
 
-exec "$@"
+/init-influxdb.sh "$@"
```

Comment:
I have added the check to execute `init-influxdb.sh` only if influxd is being run.

I also modified the scripts so that `init-influxdb.sh` gets only called with influxd arguments. The modifications made the `INFLUXDB_INIT_ONLY` environment variable obsolete since you can just run `init-influxdb.sh` instead.

I didn't modify `init-influxdb.sh` to drop a file if it runs successfully. The script already checks if `/var/lib/influxdb` is empty and will only initialize influxdb if this is the case. I dont't know if it is a good idea to remove this check.